db/migrations/202502161828_tournament_scheduled_start_and_end.up.sql

```diff
@@ -0,0 +1,10 @@
+BEGIN;
+
+ALTER TABLE  tournaments
+ADD COLUMN scheduled_start_time TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT '1970-01-01 00:00:00+00',
+ADD COLUMN scheduled_end_time TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT '1970-01-01 00:00:00+00';
```

Comment:
Defaults here basically default exclude anything created before this, which is fine. In the UI we can show a message like `No date available` is date.year < `2000`